functions/db/import_histograms.fn.js

```diff
@@ -0,0 +1,95 @@
+const admin = require('firebase-admin');
+const { BigQuery } = require('@google-cloud/bigquery');
+const functions = require('firebase-functions');
+const { murmurHash128 } = require('murmurhash-native');
+
+try {
+  admin.initializeApp();
+} catch (e) {
+  console.log(e);
+}
```

Comment:
It looks like the rest of the code in this file will run even if `admin.initializeApp()` throws an error. Would it make sense to put this `try`/`catch` block inside the export that begins on line 84?

Reply:
Good catch. I would want this cloud function to fail if there was an error here otherwise we probably couldn't talk to Firestore.

Reply:
Actually, the way I'm importing these methods from the index.js file requires this... the app may already be initialized and this just catches it if it is a 2nd initialization.